### ISSUE TITLE
Add way to skip CI for certain commits

### ISSUE THREAD

**Issue by willemneal** [near/near-sdk-rs#540]
Currently any change triggers CI tests, but adding `[ci skip]` or something similar should cause the tests not to run.  This should be a simple change to the CI setup.

**Comment by austinabell** [comment]
I'm confused, where would the ci skip be added? In the commit message? Why would we want this, if someone didn't want the CI to run for a few commits, they could just not push the commits right away?

**Comment by willemneal** [comment]
Yes it'd be in the commit message.  There are sometimes when you are making changes to say documentation, that doesn't require a CI check but it happens anyway.  The idea is to prevent unneeded executions of CI so that commits that need it won't. 

This isn't a huge issue, but I just noticed it when submitting my CODEOWNERS edit, which trigger CI and slowed down another PR's tests.

**Comment by ChaoticTempest** [comment]
@willemneal I think what you want is something like incremental tests or performing certain tests based on what changed, which might be a pretty complex machinery to add -- since this likely devolves into a dependency graph of sorts, tracking what needs to be tested

**Comment by willemneal** [comment]
Actually it might be that bad if you just test everything if any source or lock file is changed.
